_posts/en/newsletters/2022-04-13-newsletter.md

```diff
@@ -0,0 +1,193 @@
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LDK 0.0.106][] is the latest release of this LN node library.  It
+  adds support for channel identifier `alias` field proposed in [BOLTs
+  #910][] which LDK uses to enhance privacy in some cases and includes
+  several other features and bug fixes.
+
+- [BTCPay Server 1.4.9][] is the latest release of this popular payment
+  processing software.
+
+- [Bitcoin Core 23.0 RC4][] is a release candidate for the next major
+  version of this predominant full node software.  The [draft release
+  notes][bcc23 rn] list multiple improvements that advanced users and
+  system administrators are encouraged to [test][test guide] before the final release.
+
+- [LND 0.14.3-beta.rc1][] is a release candidate with several bug fixes
+  for this popular LN node software.
+
+- [Core Lightning 0.11.0rc1][] is a release candidate for the next major
+  version of this popular LN node software.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #24152][] enables [CPFP][topic cpfp] fee bumping in
+  child-with-unconfirmed-parents packages by introducing [package
+  feerate][]
+  and using it in place of individual feerate. As mentioned in
+  [newsletter #186][news186 package], this is part of a series of changes
+  to enhance the flexibility and reliability of both CPFP and [RBF][topic rbf] fee bumping.
+  This patch also [validates individual transactions first][]
+  to avoid incentive-incompatible policies like "parents pay for children"
+  or "siblings pay for siblings" behavior.
+
+- [Bitcoin Core #24098][] updates the `/rest/headers/` and
+  `/rest/blockfilterheaders/` RPCs to use query parameters for
+  additional options (e.g. `?count=<count>`) as an alternative to endpoints (e.g.
+  `/<count/`).  The documentation is updated to encourage use of the
+  query parameters over the endpoint parameters.
+
+- [Bitcoin Core #24147][] adds backend support for [miniscript][topic
+  miniscript].  Subsequent PRs [#24148][bitcoin core #24148] and
+  [#24149][bitcoin core #24149] will, if merged, add support for using
+  miniscript in [output script descriptors][topic descriptors] and in
+  the wallet's signing logic.
+
+- [Core Lightning #5165][] updates the name of the C-Lightning Project
+  to [Core Lightning][core lightning repo], or CLN for short.
+
+- [Core Lightning #5068][] adds support for attaching
```

Comment:
This "Core Lightning #5068" doesn't seem to link to a PR about adding `option_payment_metadata`. The PR it links to is about " Gossip node_announcement more aggressively, allow 2 updates per day for gossip. #5068 ". I also can't seem to find any open or closed PRs in the Core Lighting repo that mention `option_payment_metadata`. Or am I misunderstanding something?